otrv4.md

```diff
@@ -659,6 +659,12 @@ Currently, the following errors are supported:
   ERROR_2:
     Not in private state message
 ```
+Note: The string "?OTR Error:" must be in at the start position of the message because
+at least of these reasons:
+ - The possibility for playing games with the state machine by "embedding" this string
+    inside some other message.
+ - The potential of social engineering depending on how this affects the UI of the
+    chat client used.
```

Comment:
Maybe: 

"The potential of social engineering depending on the UI of the chat client used".

Reply:
(y)